### ISSUE TITLE
Ignored profile config for metrics-runtime

### ISSUE THREAD

**Issue by jean-airoldie** [metrics-rs/metrics#31]
When compiling with the toolchain version 1.35, I get the following warning:
```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /home/h4xor/forks/metrics/metrics-runtime/Cargo.toml
workspace: /home/h4xor/forks/metrics/Cargo.toml
```

This is due to the crate `metrics-runtime` which uses a [`profile config`].
However it is ignored since only a single profile set can be set per workspace (https://github.com/rust-lang/cargo/pull/3249).

Possible solutions:
* Use this as the profile for the entire workspace
* Use per package profile override (https://github.com/rust-lang/cargo/pull/3249#issuecomment-500004615)

Can't really tell which one is better since I don't understand the motivation for this profile config in the first place.

[`custom profile config`]: https://github.com/metrics-rs/metrics/blob/master/metrics-runtime/Cargo.toml#L19

**Comment by tobz** [comment]
Opened #32 to fix this.

**Comment by jean-airoldie** [comment]
Closed by 8c4e13017087daf04f095e7ae68f213c034dd5c1.
